app/views/govuk_component/breadcrumbs.raw.html.erb

```diff
@@ -33,11 +32,15 @@
           class: css_class,
           aria: {
             current: aria_current,
-          }
-        ) %>
+          },
+          itemprop: "item",
+        ) do %>
+          <span itemprop="name"><%= crumb[:title] %></span>
+        <% end %>
       <% else %>
-        <%= crumb[:title] %>
+        <span itemprop="name"><%= crumb[:title] %></span>
```

Comment:
I'm not a fan of the duplication of `span` here, but I'm not familiar enough with ERB to know the best solution so any suggests welcome!